hydra-node/test/Hydra/Chain/Direct/Contract/Mutation.hs

```diff
@@ -591,6 +592,20 @@ changeMintedValueQuantityFrom tx exclude =
  where
   mintedValue = txMintValue $ txBodyContent $ txBody tx
 
+-- | A 'Mutation' that changes the forged quantity of tokens like this:
+-- - when no value is being forged -> add some value to be forged
+-- - when tx is forging values -> add more values on top of that
```

Comment:
"Forged" is not the term we (or the cardano api/ledger etc.) would use.. it's called the mint field which can have positive (minting) or negative (burning) quantities.

Reply:
I too would have like "forge" better though.

Reply:
I though _forge_  is more general term that covers both minting and burning but I am probably wrong. Changed to mint/burn.